Dockerfile

```diff
@@ -1,7 +1,7 @@
 ########
 # assets builder and dev server
 #
-FROM node:8-slim AS assets
+FROM node:12-slim AS assets
```

Comment:
@pmac note I had to bump our node docker image here, as it was still running v8. Node 12 is the current LTS version, according to https://nodejs.org/en/.

I pushed this for a test run, and all seems to be ok? https://gitlab.com/mozmeao/www-config/pipelines/149677839

Reply:
Sounds good 👍